nixie/unittests/test_get_renderer_cmd.py

```
@@ -0,0 +1,253 @@
+        cargo_bin.mkdir(parents=True)
+        merman_path = cargo_bin / "merman-cli"
+        merman_path.write_text("")
+        merman_path.chmod(0o755)
+        monkeypatch.setattr(shutil, "which", _which_merman_only)
+
+        assert find_merman_cli() == str(merman_path)
+
+    def test_falls_back_to_path(
+        self,
+        monkeypatch: pytest.MonkeyPatch,
+        fake_home_cwd: Path,
```

Ruff (0.15.15)

[warning] Unused method argument: `fake_home_cwd`

(ARG002)